assets/js/index.js

```diff
@@ -67,47 +67,47 @@ closeClass.forEach(((element) => element.addEventListener('click', closeMenu)));
 const projects = [
   {
     name: 'Multi-Post Stories Gain+Glory',
-    languages: ['Ruby on Rails', 'Css', 'JavaScript', 'html'],
+    languages: ['Ruby on rails', 'css', 'Javascript', 'html'],
     image: '1.png',
     description: 'Lorem ipsum dolor sit amet consectetur adipisicing elit. Nesciunt quo adipisci odio maxime, officia odit distinctio? Voluptas impedit dignissimos nemo sapiente facilis cupiditate! Facere dolorum, explicabo quo eligendi eaque non?',
     livelink: '#',
     repository: '#',
   },
   {
```

Comment:
Hello @BenMukebo  Thank you for your review, but could you please let me know if you had a chance to check the assets/js/index.js file because all projects are stored as javascript object

Reply:
Hi @GOlivierNation,
Alright, let me check!!

Reply:
Nice, thanks

You can see how I called it in my js:
 <button type="button" data-index="${index}" class="success-button opne-modal">See Project</button>

on line 132

Reply:
Hope everything is clear now

Reply:
Yeah sure, Thank you @BenMukebo